### DISCUSSION TITLE
v2.16.1-beta

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#471] [General]
## What's Changed
* Migrate to a new MAC address backend yet again.
* Support for Android 14 DP2: Support using Cronet stack for API queries.
* Many bug fixes.

**Full Changelog**: https://github.com/Mygod/VPNHotspot/compare/v2.16.0...v2.16.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.16.1'>v2.16.1-beta</a>.</em>

**Comment by rzhjty** [comment]
![Screenshot_20230327_073604_be mygod vpnhotspot](https://user-images.githubusercontent.com/129014439/227812137-7412f589-381c-4854-b818-bae18cb36ce7.jpg)

![Screenshot_20230327_074137_com android settings](https://user-images.githubusercontent.com/129014439/227812251-84a836b4-8a8a-4221-9e0a-4aec5ebd70a4.jpg)

**Comment by rzhjty** [comment]
Ethernet tethering not exist？

**Reply by Mygod** [reply]
Android 11+ only.
